src/sentry/conf/server.py

```diff
@@ -4016,14 +4016,20 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
 ngrok_host = os.environ.get("SENTRY_DEVSERVER_NGROK")
 if ngrok_host and SILO_MODE != "REGION":
     SENTRY_OPTIONS["system.url-prefix"] = f"https://{ngrok_host}"
-    SENTRY_OPTIONS["system.region-api-url-template"] = ""
+    SENTRY_OPTIONS["system.region-api-url-template"] = f"https://{{region}}.{ngrok_host}"
```

Comment:
I think this will require that local development with ngrok will always require two ngrok tunnels to be setup. We should also update `src/sentry/silo/README.md` as it currently mentions that you only need ngrok on control silo.